scenarios/api.github.com/get-archive/normalized-fixture.json

```diff
@@ -21,6 +21,7 @@
       "date": "Tue, 10 Oct 2017 16:00:00 GMT",
       "expires": "Tue, 10 Oct 2017 16:00:00 GMT",
       "location": "https://codeload.github.com/octokit-fixture-org/get-archive/legacy.tar.gz/master",
+      "now": "1",
```

Comment:
this header was added by the proxy deployed to https://zeit.co/now, I’ll ignore it